client/src/services/auth.ts

```diff
@@ -0,0 +1,67 @@
+import { graphLinks } from '../graphHelper'
+import { TokenService } from './token'
+
+export class AuthService {
+    public static async login({ email, password }: { email: string; password: string }) {
+        const response = await fetch(graphLinks.login, {
+            method: 'POST',
+            body: JSON.stringify({
+                email,
+                password,
+            }),
+            headers: {
+                'Content-Type': 'application/json',
+            },
+        })
+
+        if (response.ok) {
+            const jsonResponse: {
+                token: string
+            } = await response.json()
+            TokenService.setToken(jsonResponse.token)
```

Comment:
With this changes, Login is the only way to get token now.
So as mentioned we need to implement redirect login if not login in before merging this in

Reply:
Fixed now :)